voluncheer/job_board/views.py

```diff
@@ -3,61 +3,18 @@
 
 from job_board.models import Job
 
-Organization = apps.get_model('profiles', 'Organization')
+Organization = apps.get_model("profiles", "Organization")
 
 # ======================== Job Board ============================
-test_jobs = [
```

Comment:
Why are we removing these commented out tests? Are you sure we won't use it later?

Reply:
Yes, I suppose we need to change our job models in the future, and it causes warnings in flask8